pkg/reconciler/workload/heartbeat/heartbeat_manager.go

```diff
@@ -36,27 +36,37 @@ type clusterManager struct {
 }
 
 func (c *clusterManager) Reconcile(ctx context.Context, cluster *workloadv1alpha1.WorkloadCluster) error {
+
+	defer conditions.SetSummary(
+		cluster,
+		conditions.WithConditions(
+			workloadv1alpha1.SyncerReady,
+			workloadv1alpha1.APIImporterReady,
+			workloadv1alpha1.HeartbeatHealthy,
+		),
+	)
+
 	latestHeartbeat := time.Time{}
 	if cluster.Status.LastSyncerHeartbeatTime != nil {
 		latestHeartbeat = cluster.Status.LastSyncerHeartbeatTime.Time
 	}
 	if latestHeartbeat.IsZero() {
-		klog.V(5).Infof("Marking WorkloadCluster %s|%s not ready due to no heartbeat", cluster.ClusterName, cluster.Name)
+		klog.V(5).Infof("Marking Syncer %s|%s not ready due to no heartbeat", cluster.ClusterName, cluster.Name)
```

Comment:
Maybe change the log message here so instead of talking about marking not ready, we say we're setting HeartbeathHealthy to false b/c no heartbeat?